BDArmory/Distribution/GameData/BDArmory/Localization/UI/ja.cfg

```diff
@@ -0,0 +1,1194 @@
+
+        //PilotAi GUI labels
+        #LOC_BDArmory_AIWindow_PIDAutoTuningNumSamples = A-T サンプル数
+        #LOC_BDArmory_AIWindow_PIDAutoTuningFastResponseRelevance = A-T高速応答
+        #LOC_BDArmory_AIWindow_PIDAutoTuningInitialLearningRate = A-T初期LR
+        #LOC_BDArmory_AIWindow_PIDAutoTuningSpeed = A-Tスピード
+        #LOC_BDArmory_AIWindow_PIDAutoTuningAltitude = A-T 高度
+        #LOC_BDArmory_AIWindow_PIDAutoTuningRecenteringDistance = A-T 再センター幅
+        #LOC_BDArmory_AIWindow_PIDAutoTuningFixedP = A-T固定P
+        #LOC_BDArmory_AIWindow_PIDAutoTuningFixedFields = A-T固定フィールド
+        #LOC_BDArmory_AIWindow_PIDAutoTuningClampMaximums = A-T クランプ最大値
+        #LOC_BDArmory_AIWindow_ControlLimits = コントロール
+        #LOC_BDArmory_AIWindow_ABOverrideThreshold = アフターバーナーオーバーライド
+        #LOC_BDArmory_AIWindow_SteerLimiter = ステアリミッター
+        #LOC_BDArmory_AIWindow_LowSpeedSteerLimiter = 低速リミッター
+        #LOC_BDArmory_AIWindow_LowSpeedLimiterSpeed = ローリミッター速度
+        #LOC_BDArmory_AIWindow_HighSpeedSteerLimiter = 高速リミッター
+        #LOC_BDArmory_AIWindow_HighSpeedLimiterSpeed = ハイリミッター速度
+        #LOC_BDArmory_AIWindow_AltitudeSteerLimiterFactor = Alt ステア係数
+        #LOC_BDArmory_AIWindow_AltitudeSteerLimiterAltitude = Alt ステアリング高度
+        #LOC_BDArmory_AIWindow_WaypointPreRollTime = WP プリロール時間
+        #LOC_BDArmory_AIWindow_WaypointYawAuthorityTime = WP ヨー認証時間
+        #LOC_BDArmory_AIWindow_postStallAoA = ストール後の AoA
+        #LOC_BDArmory_AIWindow_ImmelmannTurnAngle = ブラインドコーン角度の読み取り
+        #LOC_BDArmory_AIWindow_EvadeExtend = 回避/延長
+        #LOC_BDArmory_AIWindow_Evade = 回避
+        #LOC_BDArmory_AIWindow_EvasionThreshold = 距離のしきい値
+        #LOC_BDArmory_AIWindow_EvasionTimeThreshold = 時間のしきい値
+        #LOC_BDArmory_AIWindow_EvasionNonlinearity = 非線形性の回避/拡張
+        #LOC_BDArmory_AIWindow_Extend = 拡大
+        #LOC_BDArmory_AIWindow_ExtendDistanceAirToAir = 距離A2A
+        #LOC_BDArmory_AIWindow_ExtendAngleAirToAir = アングルA2A
+        #LOC_BDArmory_AIWindow_ExtendDistanceAirToGroundGuns = 長距離 A2G 砲
+        #LOC_BDArmory_AIWindow_ExtendDistanceAirToGround = 距離 A2G
+        #LOC_BDArmory_AIWindow_ExtendTargetVel = ターゲットレベル要素
+        #LOC_BDArmory_AIWindow_ExtendTargetAngle = 目標角度
+        #LOC_BDArmory_AIWindow_ExtendTargetDist = 目標距離
+        #LOC_BDArmory_AIWindow_ExtendAbortTime = 中止時間
+        #LOC_BDArmory_AIWindow_Avoidance = 血管の回避
+        #LOC_BDArmory_AIWindow_CollisionAvoidanceThreshold = 距離のしきい値
+        #LOC_BDArmory_AIWindow_CollisionAvoidanceLookAheadPeriod = 先読み時間
+        #LOC_BDArmory_AIWindow_CollisionAvoidanceStrength = 応答強度
+        #LOC_BDArmory_AIWindow_StandoffDistance = スタンドオフ距離
+        #LOC_BDArmory_AIWindow_Terrain = 地形
+        #LOC_BDArmory_AIWindow_TurnRadiusMin = 地形回避最小値
+        #LOC_BDArmory_AIWindow_TurnRadiusMax = 地形回避最大値
+        #LOC_BDArmory_AIWindow_InvertedTerrainAvoidanceCriticalAngle = 逆クリティカル。角度
+        #LOC_BDArmory_AIWindow_TerrainAvoidanceControlSurfaceDeploymentTime = Ctrl Srf 導入時間

```

Comment:
`#LOC_BDArmory_AIWindow_TerrainAvoidanceControlSurfaceDeploymentTime` is now `#LOC_BDArmory_AIWindow_TerrainAvoidanceVesselReactionTime = Vessel Reaction Time`